stl/inc/thread

```diff
@@ -73,7 +73,7 @@ private:
                                 // extern C function under -EHc. Undefined behavior may occur
                                 // if this function throws an exception. (/Wall)
         _Thr._Hnd =
-            reinterpret_cast<void*>(_CSTD _beginthreadex(nullptr, 0, _Invoker_proc, _Decay_copied.get(), 0, &_Thr._Id));
+            static_cast<void*>(_CSTD _beginthreadex(nullptr, 0, _Invoker_proc, _Decay_copied.get(), 0, &_Thr._Id));

```

Comment:
`_beginthreadex` returns `uintptr_t` which is not a pointer to unsigned int but an integer type with the same size as a pointer.